stock_quant_package_dimension/models/stock_quant_package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = 'stock.quant.package'
+
+    weight = fields.Float('Weight (kg)')
```

Comment:
:warning: `max_weight` is defined in `delivery`: https://github.com/odoo/odoo/blob/12.0/addons/delivery/models/product_packaging.py#L13 cf https://github.com/OCA/product-attribute/pull/531#pullrequestreview-326963615

Reply:
you can redefine it here, it's not really a problem.

Reply:
Actually I got messed up between the fields. Here we shouldn't use `weight` because it's a computed field: https://github.com/odoo/odoo/blob/12.0/addons/delivery/models/stock_picking.py#L28